src/main/java/ar/edu/unq/sarmiento/wicket/inscripcion/CargarInscripcionController.java

```diff
@@ -0,0 +1,272 @@
+package ar.edu.unq.sarmiento.wicket.inscripcion;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import ar.edu.unq.sarmiento.hibernate.AlumnoHome;
+import ar.edu.unq.sarmiento.hibernate.DireccionHome;
+import ar.edu.unq.sarmiento.modelo.Alumno;
+import ar.edu.unq.sarmiento.modelo.Direccion;
+
+
+@Service
+@Scope(value = ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+@Transactional
+public class CargarInscripcionController implements Serializable{
+	
+	private static final long serialVersionUID = 1L;
+
+	@Autowired
+	private AlumnoHome alumnoHome;
+	@Autowired
+	private DireccionHome direccionHome;
+	private String nombre;
+	private String genero;
+	private String dni;
+	private LocalDate fechaNacimiento;
```

Comment:
¿Y si le ponemos un DE en el medio?

```suggestion
	private LocalDate fechaDeNacimiento;
```